pom.xml

```diff
@@ -335,6 +335,16 @@
         <enabled>true</enabled>
       </snapshots>
     </pluginRepository>
+    <!-- configure the EBR plug-in repo -->
+    <pluginRepository>
+      <id>ebr</id>
+      <url>https://repo.eclipse.org/content/repositories/ebr/</url>
```

Comment:
OBR is similar to p2. EBR is also the name used by Spring Enterprise Bundle repository, which is more or less a Orbit for OSGi without p2, relying on OBR).
I believe OBR has been deprecated with OSGi R5 to replace it with another provisioning mechanism.

EBR is indeed Eclipse Bundle Recipe (as written in title of https://issues.jboss.org/browse/LOCUS-39 ) and is basically an extension to Tycho and BND to turn Maven artifacts (which aren't already OSGi-fied) as OSGI/p2 artifacts with minimal effort.
http://de.slideshare.net/guw/tasty-recipes-for-osgi-bundles